### DISCUSSION TITLE
Last updated bug? 53 years ago

### DISCUSSION THREAD

**Discussion by ariefsn** [cypress-io/cypress#25846] [General]
![image](https://user-images.githubusercontent.com/33569737/219348300-d2322358-c62f-4114-bf2a-2d8ee26f4c76.png)

Hi,
why in this part I have `53 years ago`?

Cypress 12.5.1
Ubuntu 22.04

Thanks,
Arief

**Comment by emilyrohrbough** [comment]
bug issue: https://github.com/cypress-io/cypress/issues/22188
